test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -255,7 +259,15 @@ export class DashboardPageControls extends FtrService {
     if (additionalSettings) {
       if (controlType === OPTIONS_LIST_CONTROL) {
         // only options lists currently have additional settings
-        await this.optionsListSetAdditionalSettings(additionalSettings);
+        await this.optionsListSetAdditionalSettings(
+          additionalSettings as OptionsListAdditionalSettings
+        );
+      }
+      if (controlType === RANGE_SLIDER_CONTROL) {
+        // only options lists currently have additional settings
+        await this.rangeSliderSetAdditionalSettings(
+          additionalSettings as RangeSliderAdditionalSettings
+        );
```

Comment:
We should probably remove the `// only options lists currently have additional settings` comment now 😆